server/Logger.js

```diff
@@ -78,36 +88,36 @@ class Logger {
 
   trace(...args) {
     if (this.logLevel > LogLevel.TRACE) return
-    console.trace(`[${this.timestamp}] TRACE:`, ...args)
+    console.trace(`[${this.timestamp}[ TRACE:`, ...args)
```

Comment:
This is probably not how you wanted to change the patch.

Reply:
Ah that was leftover from your original commit

Reply:
I got this on my last commit